packages/ra-ui-materialui/package.json

```diff
@@ -53,6 +53,7 @@
         "react-transition-group": "^2.2.1",
         "recompose": "~0.26.0",
         "redux": "~3.7.2",
-        "redux-form": "~7.4.0"
+        "redux-form": "~7.4.0",
+        "warning": "^4.0.2"
```

Comment:
We didn't use the `warning` package for all other warnings. I think we should either drop it or use it everywhere

Reply:
we can drop it here

Reply:
It is used on ra-core.